plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerArtifact.kt

```diff
@@ -99,6 +101,14 @@ open class CodeModernizerArtifact(
             if (!patchesDir.isDirectory()) {
                 throw RuntimeException("Expected root for patches was not a directory.")
             }
+            val patch = patchesDir.walk().map { fileSystem.findFileByNioFile(it) }.toList()
+            if (patch[0] == null) {
+                ApplicationManager.getApplication().invokeLater {
+                    ApplicationManager.getApplication().runWriteAction {
+                        VirtualFileManager.getInstance().syncRefresh() // try refreshing Virtual File System if diff.patch not found
+                    }
+                }
+            }
             return patchesDir.walk()
```

Comment:
Question: Do we need to do `patchesDir.walk()` again if the first call resulted in success?

Reply:
Perhaps not but there's only 1 file (the `diff.patch`) in that directory so IMO this is an almost negligible optimization.

Reply:
Will fix this anyway, doesn't hurt